package.json

```diff
@@ -70,7 +73,7 @@
     "serverless": "^3.26.0",
     "serverless-step-functions": "^3.12.1",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.0-dev.20220916"
+    "typescript": "^4.9.5"
```

Comment:
@devanshutanna Is this for `satisfies`?

Reply:
We did have support for satisfies already..
Just that we were using a dev build,
So updated it to the latest version.

Reply:
Understood, thanks